action.yml

```diff
@@ -24,7 +24,7 @@ inputs:
   env-regex:
     description: |
       Override the environment variables which will be passsed into the renovate container.
-      Defaults to `^(?:RENOVATE_\\w+|LOG_LEVEL|GITHUB_COM_TOKEN|NODE_OPTIONS)$`
+      Defaults to `^(?:RENOVATE_\\w+|LOG_LEVEL|GITHUB_COM_TOKEN|NODE_OPTIONS|(?:https?|no)_proxy)$`
```

Comment:
I think so, I've more commonly seen upper case. Per https://about.gitlab.com/blog/2021/01/27/we-need-to-talk-no-proxy/, it's unfortunately quite varied

Reply:
```suggestion
      Defaults to `^(?:RENOVATE_\\w+|LOG_LEVEL|GITHUB_COM_TOKEN|NODE_OPTIONS|(?:HTTPS?|NO)_PROXY|(?:https?|no)_proxy)$`
```

Reply:
We could do complete regex case insensitive with `i` flag. I don't think it will have significant impact on the other inputs , like `LOG_LEVEL` or `RENOVATE_`...